lib/withAuth.ts

```diff
@@ -58,6 +61,15 @@ export default function withAuth(handler: (req: NextApiRequestWithAuth, res: Nex
         });
       }
 
+      // Update meta data from user
+      const last_visited_ts = getTs();
+
+      await UserModel.updateOne({ _id: user._id }, {
```

Comment:
Since this is going to be called a lot, would it be possible to do a findOneAndUpdate on the user? There's a db call to get the user in the getUserFromToken function so maybe we can combine them to save a trip.

Reply:
Good idea

Reply:
done